portal/src/main/webapp/js/src/oncokb/OncoKBConnector.js

```diff
@@ -157,6 +157,7 @@ var OncoKB = (function(_, $) {
     function Instance(id) {
         this.initialized = false;
         this.dataReady = false;
+        this.civicDataReady = false;
```

Comment:
@jjgao it's ok to let civic code live inside OncoKBConnector.js for now. But no more new features should be developed, OncoKBConnector.js will soon be deprecated.